test/lib/core/utils.test.ts

```
@@ -15,46 +13,46 @@ describe('test/lib/core/utils.test.js', () => {
 
     it('should convert function', () => {
       const obj = {
         function$: function a() { console.log(a); },
-        arrowFunction$: a => { console.log(a); },
+        arrowFunction$: (a: any) => { console.log(a); },
         /* eslint object-shorthand: 0 */
-        anonymousFunction$: function(a) { console.log(a); },
-        generatorFunction$: function* a(a) { console.log(a); },
-        asyncFunction$: async function a(a) { console.log(a); },
+        anonymousFunction$: function(a: any) { console.log(a); },
+        generatorFunction$: function* a(a: any) { console.log(a); },
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)